client/my-sites/plans-v2/style.scss

```diff
@@ -0,0 +1,11 @@
+.plans-v2__columns {
+	display: flex;
+
+	& > div {
```

Comment:
It seems to be that these selectors are a bit brittle. A class (or heck, even `*`) would be safer.

Reply:
Good catch! Resolved in e8de7518491776c11c1e07f73c360e40021c8f87.